src/Refactor/Xml/NotEmptyWriter.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace DemandwareXml\Refactor\Xml;
+
+class NotEmptyWriter
+{
+    private $writer;
+
+    public function __construct(XmlWriter $writer)
+    {
+        $this->writer = $writer;
+    }
+
+    public function writeElement($name, $content = null): bool
```

Comment:
Would it be safe to type hint `$name` as a string in all of these functions? It would seem strange to accept anything else.